fe/fe-core/src/main/java/org/apache/doris/catalog/TabletStatMgr.java

```diff
@@ -43,12 +43,24 @@
 public class TabletStatMgr extends MasterDaemon {
     private static final Logger LOG = LogManager.getLogger(TabletStatMgr.class);
 
-    private ForkJoinPool taskPool = new ForkJoinPool(Runtime.getRuntime().availableProcessors());
+    private ForkJoinPool taskPool = new ForkJoinPool(Math.max(8, Runtime.getRuntime().availableProcessors()));
 
     public TabletStatMgr() {
         super("tablet stat mgr", Config.tablet_stat_update_interval_second * 1000);
     }
 
+    private ForkJoinPool adjustThreadPool(int backendSize) {
+        int minimunParallelism = Math.max(8, Runtime.getRuntime().availableProcessors());
```

Comment:
is it better?
```suggestion
        int minimunParallelism = taskPool.getParallelism();
```

Reply:
1、why not just use parallel stream for multi be task or forkJoinPool for multi be task, instead of both used ?
2、TabletInvertedIndex also has the same problem,

Reply:
@gavinchou e.x. When the be num reduce from 32 -> 8, we need to reduce the parallelism num too.

Reply:
@SWJTU-ZhangLei 
1. You are right, we should use forkJoinPool for multi be task, not use the parallel stream. I will try to fix it.
2. I will also fix it.

Reply:
And I think we need to abandon the ForkJoinPool here.

Reply:
@SWJTU-ZhangLei TabletInvertedIndex will not be modified in this commit as its ForkJoinPool is hard to be directly replaced with a regular thread pool.